packages/flutter/lib/src/widgets/scroll_notification.dart

```diff
@@ -100,7 +148,15 @@ class ScrollUpdateNotification extends ScrollNotification {
   }
 }
 
+/// A notification that [Scrollable] widget has changed its scroll position
+/// outside of its scroll bounds.
```

Comment:
actually this will fire when it _hasn't_ changed its scroll position, but would have. Basically it's what translates into a glow. On iOS this never fires.